collector/spaces.go

```diff
@@ -34,27 +35,31 @@ const spacesDomain = "%s.digitaloceanspaces.com"
 func NewSpacesCollector(logger log.Logger, errors *prometheus.CounterVec, client *godo.Client, accessKeyID string, accessKeySecret string, timeout time.Duration) *SpacesCollector {
 	errors.WithLabelValues("spaces_bucket").Add(0)
 
-	labels := []string{"creation_date", "region", "name"}
+	labels := []string{"region", "name"}
 	return &SpacesCollector{
 		logger:          logger,
 		errors:          errors,
 		client:          client,
 		timeout:         timeout,
 		accessKeyID:     accessKeyID,
 		accessKeySecret: accessKeySecret,
-
-		Active: prometheus.NewDesc(
+		Bucket: prometheus.NewDesc(
 			"digitalocean_spaces_bucket",
 			"Spaces bucket and its details. Will always be 1 if exists",
 			labels, nil,
 		),
+		BucketCreationDate: prometheus.NewDesc(
+			"digitalocean_spaces_bucket_creation_date",
```

Comment:
How about `digitalocean_spaces_bucket_created`. That would be on pair with what kube-state-metric does with creation timestamps.

Reply:
Pushed! @metalmatze